Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -75,10 +75,10 @@ class GlyphPage : public RefCounted<GlyphPage> {
 
     static unsigned count() { return s_count; }
 
-    static const unsigned size = 16;
+    static const unsigned size = 1 << 8; // 256 wold be enough for fitting all Latin-1 chars.
 
-    static unsigned sizeForPageNumber(unsigned) { return 16; }
-    static unsigned indexForCodePoint(UChar32 c) { return c % size; }
+    static unsigned sizeForPageNumber(unsigned) { return size; }
+    static unsigned indexForCodePoint(UChar32 c) { return c & 0xFF; }
```

Comment:
If someone changes `size` but not this `0xFF` things will go really bad. Would be better to write this in a way where that's not possible.